source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -5,6 +5,15 @@ Telemetrics
 
 Telemetrics in |CL-ATTR| is a client and server solution used to collect data from running |CL| systems to help quickly identify and fix bugs in the OS.  Both client and server are customizable, and an API is available on the client side for instrumenting your code for debug and analysis.
 
+.. important::
+
+   Telemetry in |CL| is **opt-in**. The telemetry client will **not** be active or send any data until you explicitly enable it.  
```

Comment:
Change 2nd sentence to be stronger:
  The telemetry client is **not** active and sends **no** data until you explicitly enable it.

Reply:
Excellent rewrite!  Changed.